kw.sh

```diff
@@ -5,9 +5,53 @@ EASY_KERNEL_WORKFLOW=${EASY_KERNEL_WORKFLOW:-"kw"}
 src_script_path=${src_script_path:-"$HOME/.config/$EASY_KERNEL_WORKFLOW/src"}
 external_script_path=${external_script_path:-"$HOME/.config/$EASY_KERNEL_WORKFLOW/external"}
 
+# Notification definitions
+declare -r ALERT_NONE=0
+declare -r ALERT_VISUAL=1
+declare -r ALERT_SOUND=2
+declare -r ALERT_BOTH=3
+
 # Export external variables required by kworkflow
 export EASY_KERNEL_WORKFLOW
 
+# Alerts command completion to the user. First argument is the alert code,
+# and second is the command string to be printed in visual notification.
+function alert_completion()
+{
+  local ALERT=$1
+  local COMMAND=$2
+  if [[ $(($ALERT & $ALERT_SOUND)) != 0 ]]; then
+    echo -e "\a\a"
+  fi
+  if [[ $(($ALERT & $ALERT_VISUAL)) != 0 ]]; then
+    notify-send -i checkbox -t 10000 "kw" "Command: \"$2\" completed!"
```

Comment:
Do we need to update the README in order to inform this dependency? This command does not work on my environment, but I suppose this is ok since I'm using openbox.
Ps.: Can we think about a terminal visual alert if "notify-send" does not work? Just any idea.